client.go

```diff
@@ -265,13 +265,15 @@ func (i *invocations) removeAll() {
 }
 
 type callbacks struct {
-	mtx  sync.Mutex
-	data map[string]*CallbackStream
+	mtx    sync.Mutex
+	config *config
```

Comment:
It's okay, but since we use the only parameter from config, maybe we don't need the full config here. Seems like the only MaxMessageProcessDuration will be enough here and our code will be more clear and straightforward.
What do you think?